File: DayTwo/Java/LevelOne/TaskTen.java
import java.util.Scanner;

public class taskThree{
    public static void main(String... args){
        
    Scanner input = new Scanner(System.in);
    
    System.out.print("Enter the first number: ");
    int numberOne = input.nextInt();
    
    System.out.print("Enter the second number: ");
    int numberTwo = input.nextInt();
    
    System.out.print("Enter the second number: ");
    int numberThree = input.nextInt();
    
    int average = (numberOne + numberTwo + numberThree) / 3;
    
    System.out.printf("The average = %d", average);
    }
}
   

File: DayTwo/JavaScript/LevelOne/taskTen.js
const prompt = require("prompt-sync")();

let numberOne = Number(prompt("Enter the first number: "));
let numberTwo = Number(prompt("Enter the second number: "));
let numberThree = Number(prompt("Enter the third number: "));

let average = (numberOne + numberTwo + numberThree) / 3;

console.log(`The average = ${average}`);
